torch/__init__.py

```diff
@@ -1489,3 +1489,10 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
                   'use torch.sparse_coo_tensor(..., check_invariants=False) instead.')
     kwargs['check_invariants'] = False
     return torch.sparse_coo_tensor(*args, **kwargs)
+
+# Trigger registration of functional distributed collectives
+# This is needed by torch_deploy
+try:
+    import torch.distributed._functional_ops as funops
+except BaseException as _:
```

Comment:
So this isn't gonna be an ImportError? why BaseException

Reply:
Don't suppress like this. Figure out the condition in which this module can be imported and do it that way

Reply:
👍

Reply:
Removed the import from `torch`